recipes/mirfix/meta.yaml

```diff
@@ -16,13 +16,15 @@ build:
 requirements:
   run:
     - python >=3
-    - biopython
-    - tk
-    - numpy
-    - matplotlib-base
-    - viennarna
-    - dialign2
-    - clustalw
+    - biopython =1.78
```

Comment:
are the strict pins really needed? Less strict is always better :)

Reply:
Unfortunately yes